.gitignore

```diff
@@ -6,6 +6,7 @@ bazel-*
 
 test_temp/
 build/
+cmake-build-debug/
```

Comment:
I don't think this is necessary. To avoid generation of this directory, you can configure build directory of CMake profile to `build` in CLion.